internal/cortex/chunk/chunk.go

```diff
@@ -353,10 +353,10 @@ func (c *Chunk) Slice(from, through model.Time) (*Chunk, error) {
 	return &nc, nil
 }
 
-func intervalsOverlap(interval1, interval2 model.Interval) bool {
-	if interval1.Start > interval2.End || interval2.Start > interval1.End {
-		return false
-	}
+// func intervalsOverlap(interval1, interval2 model.Interval) bool {
```

Comment:
Can we remove this? :smile: why is this commented?

Reply:
Leftover from my initial attempt of what can be removed. Deleted and rebased on main.